VictoryCenter/VictoryCenter.WebAPI/.env.example

```
@@ -1,14 +1,14 @@
-BLOB_LOCAL_STORE_KEY=my-super-secret-local-key
-WAY4PAY_MERCHANT_LOGIN=your-merchant-login
-WAY4PAY_MERCHANT_SECRET_KEY=your-merchant-secret-key
-WAY4PAY_MERCHANT_DOMAIN_NAME=https://example.com
-WAY4PAY_API_URL=https://secure.wayforpay.com/pay
-CLOUDFLARE_TURNSTILE_SECRET_KEY=your-cloudflare-turnstile-secret-key
-CLOUDFLARE_TURNSTILE_SITE_VERIFY_URL=https://challenges.cloudflare.com/turnstile/v0/siteverify
+DB_CONNECTION_STRING="server=your_db_server;Database=your_db_name;User Id=your_db_user;Password=your_db_password;MultipleActiveResultSets=true;TrustServerCertificate=true;Encrypt=false;"
+INITIAL_ADMIN_EMAIL=your_admin_email@example.com
+INITIAL_ADMIN_PASSWORD=your_admin_password
+JWTOPTIONS_SECRETKEY=your_jwt_access_token_secret_key
+JWTOPTIONS_REFRESH_TOKEN_SECRETKEY=your_jwt_refresh_token_secret_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWTOPTIONS_REFRESH_TOKEN_SECRETKEY key should go before the JWTOPTIONS_SECRETKEY key

(UnorderedKey)